### PR TITLE
Fix Part of #2833 : Add missing named argument in CompletedStoryListActivityTest

### PR DESCRIPTION
**Pull request by techjd**
## Explanation

  - Fixes Part of oppia#2833 Added Missing Named Argument in CompletedStoryListActivityTest
  

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt [hunk 1]
```diff
@@ -145,7 +145,7 @@ class CompletedStoryListActivityTest {
       onView(
         atPositionOnView(
           R.id.completed_story_list,
-          0,
+          position = 0,
           R.id.completed_story_lesson_thumbnail
```
Comment (prayutsu) [inline app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt]: @anandwana001 Do we want named arguments for only `position` parameter or we should have named arguments for all the three parameters here?

**Review by prayutsu — COMMENTED** [review]
@techjd PR looks good, just wait for @anandwana001's reply for my above comment.

**Comment by prayutsu** [comment]
@techjd Also please remove the word "oppia" from the PR title.

**Comment by prayutsu** [comment]
@techjd One more change, as this PR fixes only a part of the main issue. You should change the title and PR description to `Fix part of #2833: <PR title> ` and `Fixes part of #2833: <Explanation>` respectively.

File: app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt]: All we need for consistency and it looks good too.

Comment (prayutsu) [reply app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt]: @techjd You can go ahead and do the same for the other 2 parameters also.

Comment (techjd) [reply app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt]: @prayutsu I have changed for other 2 parameters . PTAL

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Thanks, @techjd Looks good so far. 
Please look for more missing arguments like at line number 114.

**Comment by techjd** [comment]
@anandwana001 Line 114 and one more missing argument did came into my mind , but then after changing it's becoming profileId = profileId , which was looking quite absurd to me . Should I still change it ?

**Comment by anandwana001** [comment]
we can do this for consistency as the second argument is having a name too.

**Comment by techjd** [comment]
@anandwana001 Some more missing arguments are added . PTAL

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @techjd

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**